src/imgui_ex/ImGuizmo.cpp

```diff
@@ -148,7 +148,9 @@ static ImVec2 vcGizmo_WorldToScreen(const udDouble3& worldPos, const udDouble4x4
   if (trans.w != 0)
     trans *= 0.5f / trans.w;
   trans += udDouble4::create(0.5, 0.5, 0.0, 0.0);
+#if GRAPHICS_API_D3D11
```

Comment:
Wouldn't this be a `#if !GRAPHICS_API_OPENGL` operation? Metal and Vulkan would likely need this too?